lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -63,6 +63,8 @@ def cmd_run(*args, action: nil)
       return false
     end
 
+    driver.run_single('reload_lib -a') if args[:reload_libs]
```

Comment:
Can we move this up higher, otherwise it's not possible to reload the library without having valid module options